fe/fe-core/src/main/java/org/apache/doris/persist/AlterLSCOperationLog.java

```diff
@@ -0,0 +1,102 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.persist;
+
+import org.apache.doris.common.io.Text;
+import org.apache.doris.common.io.Writable;
+import org.apache.doris.persist.gson.GsonUtils;
+import org.apache.doris.proto.InternalService.PFetchColIdsResponse;
+import org.apache.doris.proto.InternalService.PFetchColIdsResponse.Builder;
+import org.apache.doris.proto.InternalService.PFetchColIdsResponse.PFetchColIdsResultEntry;
+
+import com.google.common.base.Preconditions;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class AlterLSCOperationLog implements Writable {
+
+    @SerializedName(value = "dbId")
+    private Long dbId;
+
+    @SerializedName(value = "tableId")
+    private Long tableId;
+
+    @SerializedName(value = "indexIds")
+    private List<Long> indexIds;
+
+    @SerializedName(value = "colNameToIds")
```

Comment:
how about  put  `indexIds` and `colNameToIds` into a map? don't use PFetchColIdsResponse here